lib/automatic-new-tag.js

```diff
@@ -0,0 +1,37 @@
+const BroccoliPlugin = require('broccoli-plugin');
+const walkSync = require('walk-sync');
+const { readFileSync, writeFileSync } = require('fs');
+const { join } = require('path');
+const yamlFront = require('yaml-front-matter');
+const yaml = require('js-yaml');
+
+module.exports = class AutomaticNewTag extends BroccoliPlugin {
+  constructor(folder, options) {
+    super([folder], options);
+  }
+
+  build() {
+    this.inputPaths.forEach((dir) => {
+
+      const markdownFiles = walkSync(dir)
+        .filter(path => path.endsWith('.md'));
+
+      markdownFiles.forEach((file, index) => {
+        const fileContents = readFileSync(join(dir, file))
+        const frontMatter = yamlFront.loadFront(fileContents);
+
+        if (index < 4) {
```

Comment:
so this is probably the right way to do this... although I think we might need to sort the files based on date first 🤔 there is a strong possibility that the current test is only passing because the files are in alphabetical order as well as chronological order 🤔